WordPressKit/CommentServiceRemoteXMLRPC.m

```diff
@@ -117,6 +117,7 @@ - (void)moderateComment:(RemoteComment *)comment
                 failure:(void (^)(NSError *))failure
 {
     NSParameterAssert(comment.commentID != nil);
+    NSNumber *commentID = comment.commentID;
     NSArray *extraParameters = @[
                                  comment.commentID,
```

Comment:
One tiny suggestion.  Maybe use `commentID` here as well rather than going through the `comment` instance as it currently does? This would make how the comment ID was referenced consistent throughout this method.